code/drasil-website/lib/Drasil/Website/Example.hs

```diff
@@ -208,15 +208,18 @@ getSRSRef path sufx ex = makeURI refUID (getSRSPath path (map toLower sufx) ex)
 -- | Get the paths of where each reference exist for SRS files. Some example abbreviations have spaces,
 -- so we just filter those out. The suffix should only be either html or pdf.
 getSRSPath :: FilePath -> String -> String -> FilePath
-getSRSPath path sufx ex = path ++ map toLower (filter (not.isSpace) ex)
-  ++ "/SRS/srs/" ++ filter (not.isSpace) ex ++ "_SRS." ++ map toLower sufx
+getSRSPath path sufx ex = 
+  path
+  ++ map toLower ex -- FIXME: The majority of these `map toLower`s are implicit knowledge!!! 
```

Comment:
The `map toLower`s happen quite often!

Reply:
Do these show up as a redundancy to enforce them being lowercase, or is there something that is being made lowercase in multiple places (instead of just being made lowercase at its definition)?

Reply:
Ah, that's a good catch. I completely forgot to address why they appear.

It does not appear to be a redundancy to enforce them being lowercase. I don't really think we should _need_ nearly any sort of redundancy, since we generate through staging.

These `map toLower`s applied to our case study names is to relate to the things that were named after our project folder names in the `code/drasil-example` folder -- they are all lowercase. We should probably figure out a better way of handling external resources internally, preferrably reading them in externally and having an external dictionary that's passed to/compiled into Drasil.

Reply:
I feel like this could be related to how we've been handling paths to Projectile-related files (since it uses design variabilities that introduce another directory level) and figuring out how to handle this in the long term 🤔

Reply:
Yep, it definitely is. We might need to do another pass at cleaning this up.

Reply:
Also related to #3557